scripts/nodes/n0_test

```diff
@@ -25,7 +25,6 @@ rm /tmp/main.*.log
   --testnet=true  \
   --api-port=18001 \
   --protocol-port=18000 \
```

Comment:
Can we delete the whole nodes folder here? People should only use the ones in backend from now on. These are outdated and misleading.

Reply:
This makes sense to me 👍